CHANGELOG.md

```diff
@@ -64,9 +77,32 @@ Those changes in added, changed or breaking changes, should include usage exampl
     .status(); 
   // tx_status should be true
   ```
-
-* Added custom configuration for release-plz in [#281](https://github.com/Layr-Labs/eigensdk-rs/pull/281).
 * Added Rewards2.1 support in [#323](https://github.com/Layr-Labs/eigensdk-rs/pull/323).
+  * set_operator_set_split
+  ```rust
+      let operator_set = OperatorSet {
+            avs: avs_address,
+            id: 0,
+        };
+
+        let new_split = 5;
+        let tx_hash = el_chain_writer
+            .set_operator_set_split(OPERATOR_ADDRESS, operator_set.clone(), new_split)
+            .await
+            .unwrap();
+  ```
+  * get_operator_set_split
+  ```rust
+     let operator_set = OperatorSet {
+            avs: avs_address,
+            id: 0,
+        };
+       let split = el_chain_writer
+            .el_chain_reader
+            .get_operator_set_split(OPERATOR_ADDRESS, operator_set)
+            .await
+            .unwrap(); 
+  ```
```

Comment:
Nit: let's add some spaces and improve the 

`````suggestion

  * Set an operator's split on an operator set.

  ```rust
      let operator_set = OperatorSet {
            avs: avs_address,
            id: 0,
        };

        let new_split = 5;
        let tx_hash = el_chain_writer
            .set_operator_set_split(OPERATOR_ADDRESS, operator_set.clone(), new_split)
            .await
            .unwrap();
  ```

  * Get an operator's split on an operator set.

  ```rust
     let operator_set = OperatorSet {
            avs: avs_address,
            id: 0,
        };
       let split = el_chain_writer
            .el_chain_reader
            .get_operator_set_split(OPERATOR_ADDRESS, operator_set)
            .await
            .unwrap(); 
  ```

`````